src/OracleService/Protocols/OracleNeoFSProtocol.cs

```diff
@@ -102,7 +102,7 @@ private static async Task<string> GetRangeAsync(Client client, Address addr, str
         private static async Task<string> GetHeaderAsync(Client client, Address addr, CancellationToken cancellation)
         {
             var obj = await client.GetObjectHeader(cancellation, new ObjectHeaderParams() { Address = addr }, new CallOptions { Ttl = 2 });
-            return obj.Payload.ToString(Utility.StrictUTF8);
+            return obj.ToString();
```

Comment:
Where is the definition of the ToString() method?

Reply:
https://github.com/neo-ngd/neofs-api-csharp/blob/055f93206e374ed6c30fa7865273b6cda9c9c548/src/Neo.FileStorage.API/object/Types.cs#L1848-L1851

It's auto-generated.

Reply:
> return pb::JsonFormatter.ToDiagnosticString(this);

Is it deterministic? Maybe we should reads the content and use `JObject` in neo core.

Reply:
Will do. BTW, `obj` is a instance of neofs `Object` class with empty `Payload`. We need transfer it to JSON.

Reply:
Done.